src/MergeContext.js

```diff
@@ -787,9 +787,24 @@ class PullRequest {
         return (this._rawPr.title + ' (#' + this._rawPr.number + ')' + '\n\n' + this._prBody()).trim();
     }
 
+    // Searches for the first non_ASCII_printable character and returns
+    // its position in the string. By default, JavaScript strings are
+    // represented as a sequence of UTF-16 characters.
+    _invalidCharacterPosition(str) {
+        const prohibitedCharacters = /[^\u0020-\u007f]+/; // allow non-special ASCII characters
```

Comment:
The code probably "works", but, AFAICT, we should be using `\u{}` characters and ES6 [regexes](https://mathiasbynens.be/notes/es6-unicode-regex) here. They are safer, especially when it comes to [character classes](https://mathiasbynens.be/notes/es6-unicode-regex#impact-character-classes).

Reply:
I don't know why, but this code point escapes did not work in my tests (node v9). Moreover, I decided to use `\x` for matching, since it has the same effect and is shorter.

Reply:
> I don't know why, but this code point escapes did not work in my tests (node v9)

Just for the record, did you use the `/u` regex flag when testing `\u{}` code point expressions?

Reply:
Well, I see: I looked [here](https://mathiasbynens.be/notes/javascript-escapes#unicode-code-point) when learning the syntax, where '/u' is not mentioned. Now it works.

Reply:
Right, that section talks about usage in strings, not regexes.

Reply:
> Now it works

... but the code is still missing `/u` flag for the regex. AFAICT, that may even be a source of a bug if our strings may contain Unicode:

> The u flag also affects negated character classes. For example, /[^a]/ is equivalent to /[\0-\x60\x62-\uFFFF]/, which would match any BMP symbol except a. But with the u flag, /[^a]/u matches the much bigger set of all Unicode symbols except a.

Please also use explicit brackets for Unicode points and, if possible, avoid leading zeros (e.g., `\u{20}`).

Reply:
Done at 4492ab51.